go/cmd/vtctl/vtctl.go

```diff
@@ -100,7 +100,7 @@ func main() {
 		log.Warningf("cannot connect to syslog: %v", err)
 	}
 
-	if err := reparentutil.SetDurabilityPolicy(*durabilityPolicy); err != nil {
+	if err := reparentutil.SetDurabilityPolicy("none"); err != nil {
```

Comment:
Why do we even need to set this?

Reply:
We need to set some durability policy so that the code does not panic. Since we want to remove the flag, we need to set something. I am going to refactor this in an upcoming PR, where we first check the keyspace information for the durability_policy and we set it there.

Reply:
Ok, that's fine.